public/index.php

```diff
@@ -17,11 +17,11 @@
 require __DIR__.'/../vendor/autoload.php';
 
 // The check is to ensure we don't use .env in production
-if (!getenv('APP_ENV')) {
+if (!isset($_SERVER['APP_ENV'])) {
     (new Dotenv())->load(__DIR__.'/../.env');
 }
 
-if (getenv('APP_DEBUG')) {
+if ($_SERVER['APP_DEBUG'] ?? true) {
```

Comment:
this looks broken, as setting it to `'0'` will still be a truthy string.

Reply:
and `'0' == false` is true